docs/manual/wikibase/configuration.md

```diff
@@ -35,7 +35,7 @@ aware of anyone planning to work on this, though.
 
 The manifest is a JSON object describing all the configuration details necessary for OpenRefine to integrate with your Wikibase instance. As an example, here is the manifest of Wikimedia Commons:
 
-```json
+````json
```

Comment:
What is the difference between starting a code block with four backquotes compared to just three?

Reply:
Here, I am sorry. It come's to my practice of doing this because some I use to do it in md files, I will revert it to ``` column only then

Reply:
I rectified it this time. Thank you and sorry for this feedback.